src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -1294,6 +1297,12 @@ public override BoundNode VisitEventAssignmentOperator(BoundEventAssignmentOpera
         }
 
         protected virtual void VisitArguments(ImmutableArray<BoundExpression> arguments, ImmutableArray<RefKind> refKindsOpt, MethodSymbol method)
+        {
+            VisitArgumentsBeforeCall(arguments, refKindsOpt);
```

Comment:
>VisitArgumentsBeforeCall(arguments, refKindsOpt); [](start = 12, length = 49)

Please add an assert that method is not a local function. It also would be good to add a doc comment about that requirement.